kevinene91/Store-Manager-API
# #161209990 tests for API

### What does this PR do? 
It has all the tests for the various endpoints for the app includes:
- auth endpoints 
- category endpoints 
- product endpoints 
- sales endpoints 

### Description of Task completed?
Have tests for the API endpoints 

### How to Manually test this?
 
- Clone the repository `git clone https://github.com/kevinene91/Store-Manager-API.git`
- Change into the app directory `cd Store-Manager-API`
- get this branch code `git fetch origin ch-tests-161209990`
- Navigate the folder structure
- create a virtual environment `python3 -m pyvenv venv`
- pip install -r requirements.txt
-  run `pytest` 47 tests should fail 

### Relevant PVT story? 

[161209990](https://www.pivotaltracker.com/n/projects/2203367/stories/161209990)
